packages/web-components/src/menu-item/menu-item.styles.ts

```diff
@@ -0,0 +1,134 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+  borderRadiusMedium,
+  colorCompoundBrandForeground1Hover,
+  colorCompoundBrandForeground1Pressed,
+  colorNeutralBackground1,
+  colorNeutralBackground1Hover,
+  colorNeutralBackground1Pressed,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForeground2,
+  colorNeutralForeground2Hover,
+  colorNeutralForeground2Pressed,
+  colorNeutralForeground3,
+  colorNeutralForegroundDisabled,
+  colorNeutralStrokeDisabled,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase500,
+  fontWeightRegular,
+  fontWeightSemibold,
+  lineHeightBase200,
+  lineHeightBase300,
+} from '../theme/design-tokens.js';
+
+/** Menu styles
+ * @public
+ */
+export const styles = css`
```

Comment:
Focus frame is cropped:
![image](https://user-images.githubusercontent.com/9615899/220403238-d4f1ba14-4ca3-4d6d-9076-edd61129d1d5.png)

And does not match FUIR9 styles:
<img width="170" alt="image" src="https://user-images.githubusercontent.com/9615899/220403433-00323957-2a35-4ff0-ab99-09f5a430cc82.png">

Reply:
fixed:

![image](https://user-images.githubusercontent.com/11062709/220455416-2d57021f-226d-4273-928c-13726f1c11e7.png)